scripts/BuildDocs.ps1

```diff
@@ -11,18 +11,41 @@ $docsRepoPath = [System.IO.Path]::GetFullPath("$PSScriptRoot\..\docs\_repo")
 # Ensure the tools path exists
 mkdir $toolsPath -Force | Out-Null
 
+if (![System.IO.File]::Exists($docfxBinPath)) {
+    mkdir $docfxBinPath -Force | Out-Null
+}
+
 if (![System.IO.File]::Exists($docfxExePath)) {
     # Download DocFX
-    Remove-Item -Path "$docfxBinPath" -Force -ErrorAction Stop | Out-Null
-    (new-object net.webclient).DownloadFile('https://github.com/dotnet/docfx/releases/download/v1.8/docfx.zip', "$docfxZipPath")
+    if (Test-Path -Path $docfxBinPath) { Remove-Item -Path "$docfxBinPath" -Force -ErrorAction Stop | Out-Null }
+    # Github uses TLS 1.2
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```

Comment:
It is better to only add `Tls12` if needed instead of changing it to `Tls12` only. Also, please use try-finally to restore the original securityProtocolType.
```suggestion
    [Net.ServicePointManager]::SecurityProtocol = [Net.ServicePointManager]::SecurityProtocol -bor [Net.SecurityProtocolType]::Tls12
```

Reply:
Used `-bor`